### PR TITLE
Fuzzing sigstore

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Included sigstore/sigstore project

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@oliverchang FYI...

File: projects/sigstore/project.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+homepage: "https://sigstore.dev"
+main_repo: "https://github.com/sigstore/sigstore"
+primary_contact: "neevan.srinivasan@gmail.com"
```
[the remark this answers is not in the document]
Comment (naveensrinivasan) [reply projects/sigstore/project.yaml:3]: I agree, I will update the PR once it is working. Up until then I will have my email address.

[the remark this answers is not in the document]
Comment (DavidKorczynski) [reply projects/sigstore/project.yaml:3]: OSS-Fuzz does not support encrypting the content of bug reports

File: projects/sigstore/build.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+fuzzer=fuzz_loadCertificates.a
+cd "$GOPATH"/src/github.com/sigstore/sigstore/test/fuzz/pem
+
+go-fuzz -tags go-fuzz -func FuzzLoadCertificates -o $fuzzer  .
+
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer -o $OUT/$fuzzer
```
Comment (oliverchang) [inline projects/sigstore/build.sh:22]: nit: you can just use the `compile_go_fuzzer` script instead (https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/#buildsh)

File: projects/sigstore/project.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (oliverchang) [reply projects/sigstore/project.yaml:3]: Lets add at least one other person from sigstore here. These emails are not notified unless there is a new bug report (or a build failure).

**Review by oliverchang — APPROVED** [review]
nice!

File: projects/sigstore/build.sh (hunk 2 above)
Comment (naveensrinivasan) [reply projects/sigstore/build.sh:22]: I am having trouble with compiling with it. TBH, I don't mind if you can fix it and push it. I can use the same for sigstore/sigstore ClusterFuzzLite.

File: projects/sigstore/project.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (naveensrinivasan) [reply projects/sigstore/project.yaml]: I agree, Thanks. Took care of it.

File: projects/sigstore/build.sh (hunk 2 above)
Comment (oliverchang) [reply projects/sigstore/build.sh]: I've fixed up your PR to use `compile_go_fuzzer`.

Comment (naveensrinivasan) [reply projects/sigstore/build.sh]: Thank you!

**Comment by jonathanmetzman** [comment]
If we're fuzzing sigstore in OSS-Fuzz, then I think the PR adding ClusterFuzzLite to sigstore is wrong.
OSS-Fuzz users don't need to do another integration for ClusterFuzzLite. They should use CIFuzz (which has the same functionality) instead.

**Comment by naveensrinivasan** [comment]
> If we're fuzzing sigstore in OSS-Fuzz, then I think the PR adding ClusterFuzzLite to sigstore is wrong.
> OSS-Fuzz users don't need to do another integration for ClusterFuzzLite. They should use CIFuzz (which has the same functionality) instead.

OK, that Makes sense. I will remove the PR in sigstore to use the clusterfuzzlite. Thanks

**Comment by jonathanmetzman** [comment]
> > If we're fuzzing sigstore in OSS-Fuzz, then I think the PR adding ClusterFuzzLite to sigstore is wrong.
> > OSS-Fuzz users don't need to do another integration for ClusterFuzzLite. They should use CIFuzz (which has the same functionality) instead.
> 
> OK, that Makes sense. I will remove the PR in sigstore to use the clusterfuzzlite. Thanks

Yes. You can just use the instructions here to set up CIFuzz: https://google.github.io/oss-fuzz/getting-started/continuous-integration/

**Comment by naveensrinivasan** [comment]
How do we coverage for oss-fuzz?

Also for CIFuzz if there is crash will everyone be able to see the crash call-stack and the data that caused it to happen?

**Pull request merged**